corehq/apps/userreports/templates/userreports/partials/link_reports.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+<button type="button"
+        data-toggle="modal"
+        data-target="#link-reports"
+        class="btn btn-default">
+  <i class="fa fa-link"></i> {% trans "Copy Report" %}
+</button>
+
+<div class="modal fade" id="link-reports">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h3>
+          {% trans "Copy Report" %}
+        </h3>
+      </div>
+      <div class="modal-body">
+        {% if copy_report_form %}
+          <div id="copy-report-form">
+            <form class="form form-horizontal" method="post" action="{% url "copy_report" domain %}">
+              {% crispy copy_report_form %}
+            </form>
+          </div>
+        {% endif %}
+      </div>
+      <div class="modal-footer">
```

Comment:
Can you make the form encompass the whole modal, remove the submit from `CopyReportForm`, and stick a submit button here? It's weird to have the shaded form actions and submit button inside the modal.

Given that this form only has one input, crispy isn't doing much for you.

Reply:
Removed the form and added this as an HTML form.